File: 4116-MinimumMovesToEqualArrayElementsIii/4116-MinimumMovesToEqualArrayElementsIii.java
// Last updated: 31/01/2026, 07:57:37
class Solution {
    public int minMoves(int[] nums) {
        Arrays.sort(nums);
        int sum=0;
        for(int i=0;i<nums.length;i++){
            int a=nums[nums.length-1]-nums[i];
            sum=sum+a;
        }
        return sum;
    }
}

File: 4116-MinimumMovesToEqualArrayElementsIii/4116-MinimumMovesToEqualArrayElementsIii.py
# Last updated: 31/01/2026, 07:57:35
class Solution(object):
    def minMoves(self, nums):
        nums.sort()
        sum1=0
        for i in range(0,len(nums)):
            a=nums[len(nums)-1]-nums[i]
            sum1+=a
        return sum1